transformer/dockerfilegenerator/dotnetcoredockerfilegenerator.go

```diff
@@ -420,8 +398,11 @@ func (t *DotNetCoreDockerfileGenerator) Transform(newArtifacts []transformertype
 			logrus.Errorf("Could not read the project file (%s) : %s", filepath.Join(a.Paths[artifacts.ProjectPathPathType][0], csprojRelFilePath), err)
 			continue
 		}
-		dotNetCoreTemplateConfig.DotNetVersion, err = getDotNetCoreVersion(filepath.Join(t.Env.GetEnvironmentContext(), "mappings/dotnetcoreversions.yaml"), csprojConfiguration.PropertyGroup.TargetFramework)
-		if err != nil || dotNetCoreTemplateConfig.DotNetVersion == "" {
+		frameworkVersion := dotnetcoreRegex.FindAllStringSubmatch(csprojConfiguration.PropertyGroup.TargetFramework, -1)
+		if len(frameworkVersion[0]) == 2 {
```

Comment:
This will panic if framework version is an empty array. Wouldn't it be better to check is length too?

Reply:
Yes, I was just updating it. Pushed the changes now.